### DISCUSSION TITLE
how can i unblock CORS policy

### DISCUSSION THREAD

**Discussion by Ezugo23** [mockapi-io/docs#13] [Q&A]
i am trying to create an employee website that can edit, delete and update employee but the edit is not working it keeps telling me Cors policy has been blocked what can i do i am a react developer

**Comment by javierCRC** [comment]
I have exactly the same problem with the cors policies.

**Comment by nsmirn0v** [comment]
CORS is allowed for all requests. I just tried it and it works for me. Can you share how you calling your mock API?
